### DISCUSSION TITLE
Booking apiRest full for free or cheap

### DISCUSSION THREAD

**Discussion by bertikal** [community/community#28732] [Apps, API and Webhooks]
Hi there,

Nice to talk to community. I would like to ask a question in general for my new app. I am trying to make an app like a hotel booking one. The question is that i am still looking for a free-cheap plan api where i can get hotel list, detailed hotel list, and the main thing is to get the way where the customer can go though or link to pay for the best price in, like any other booking app. Otherwise, if i can only show up details and hotels, the app can be useless for customers. Thanks

**Comment by BergerAPI** [comment]
One way would be to scrape some Hotel-Web-Pages via a cron-job and enter the prices into a database. This way the data comes from you, and is completely free. Obviously this is more work than just using an API, but you're ore independent and don't require another service to run. Let the cron-job run every 12 hours and you should always have the best prices.

**Comment by bertikal** [comment]
Thanks so much for the answer, but unfortunately that is so much work for me, I prefer something easier no matter if I have to pay for a good monthly plan( cheaper, less than 30$).
